components/app_bar/app_bar.test.tsx

```diff
@@ -112,4 +112,14 @@ describe('components/app_bar/app_bar', () => {
 
         expect(wrapper).toMatchSnapshot();
     });
+
+    test('should match snapshot on mount when App Bar is disabled', async () => {
+        mockState.entities.general.config.EnableAppBar = 'false';
```

Comment:
Even though this is the last test in the file, can we edit this config without mutating the value shared among the tests?

Reply:
It's actually reset in the `beforeEach` statement, so every test starts with the same state.

Reply:
This isn't necessarily something we need to change here since this is just test code for a single file, but mutating state like this feels bad to me. It works here because we're resetting `mockState` between test cases, but as soon as we're doing this more than once in a test case (like if we're seeing how a change in redux state affects a component), we're going to run into selectors failing to recompute because they don't know it changed.